mlbgame/data.py

```diff
@@ -1,10 +1,22 @@
+#!/usr/bin/env python
+
 '''
 This module gets the XML data that other functions use.
 It checks if the data is cached first, and if not, 
 gets the data from mlb.com
 '''
+
+from __future__ import print_function
+
+import sys
 import os
-import urllib2 as url
+
+if sys.version_info[0] == 2:
+    from urllib2 import urlopen
+else:
+    from urllib.request import urlopen
```

Comment:
Using an if here is sort of unusual. It seems far more common to use a try: except: for this. 
```python
try:
    from urlib.request import urlopen
except:
    from urllib2 import urlopen
```

The reason being that it won't rely on sys and a try is usually faster, at least, it is provided you take the try.
It is up to you which case you think belongs in the try though. My team usually puts the Python 3 code there only since it will continue to gain popularity so we figure we don't want to flip it later. 

If you are interested, the difference isn't insignificant. This was run against 3.5
```python
>>> timeit.timeit("""if sys.version_info[0] > 2: from urllib.request import urlopen
... else: from urllib2 import urlopen""")
1.7361162674080004
>>>
>>> timeit.timeit("""try: from urllib.request import urlopen
... except ImportError: from urllib2 import urlopen""")
1.185438128836065
```